common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -540,4 +545,159 @@ public Optional<String> getIconId() {
             });
         }
     }
+
+    private static final class SeaskipperDestinationCategory implements MapCategory {
+        @Override
+        public String getCategoryId() {
+            return "wynntils:seaskipper-destination";
```

Comment:
Maybe we should be a bit proactive and call this `wynntils:fast-travel:seaskipper-destination`, so we can group it with other fast travel destinations?